### ISSUE TITLE
Allow sorting by best quality orders

### ISSUE THREAD

**Issue by K-Ho** [okwme/0xchange#4]
Possibly add a column for "score" where we rank the quality of the order:  Arbitrage opportunity is ranked 100/100.  otherwise, score = (offer price / market price) * 100

**Comment by okwme** [comment]
good idea, but we'd have to precompile the score based on a market price that would be constantly shifting. may be good to update them once a day or once an hour?
